### PR TITLE
Fixes for tchellomello/python-ring-doorbell#162

### PR DESCRIPTION
**Pull request by ZachBenz**
Compatibility with Python 2 (old-school typing syntax in docstrings); fix for OAuth.SCOPE - changed to a list to match what oauthlib expects (credit: @steve-gombos)

Fixes: #162

### CODE REVIEW

**Comment by ZachBenz** [comment]
@tchellomello Lint issues addressed

File: ring_doorbell/__init__.py [hunk 1]
```diff
@@ -32,11 +32,13 @@ class Ring(object):
     """A Python Abstraction object to Ring Door Bell."""
 
     def __init__(self, username, password,
-                 auth_callback: Callable[[], str] = None,
+                 auth_callback = None,
```
[the remark this answers is not in the document]
Comment (tchellomello) [reply ring_doorbell/__init__.py]: @ZachBenz  could you please address this one too?

**Comment by ZachBenz** [comment]
> @ZachBenz one more fix and we should be ready to merge. Could you please address that for us?

Will do!  Sorry about that.

**Comment by ZachBenz** [comment]
@tchellomello Hopefully that did it!

**Pull request merged**